spectree/config.py

```diff
@@ -27,7 +37,7 @@ class Contact(BaseModel):
     #: contact url
     url: Optional[AnyUrl] = None
     #: contact email address
-    email: Optional[EmailStr] = None
+    email: Optional[EmailFieldType] = None
```

Comment:
I wanted to make the docs show `str` for the type of the email field, but couldn't figure out how to override it besides the following:

1. Don't have `email-validator` installed while generating docstrings
2. Use [`autodoc_type_aliases`](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_type_aliases), but that would require `from __future__ import annotations`

Thoughts? I'm not experienced with sphinx, so maybe there's a better way. In any case, it's not a big deal if it still shows as `EmailStr` I guess.

Reply:
I don't have a better solution. I'm okay with it.